src/redux/actions/category.action.js

```diff
@@ -0,0 +1,15 @@
+import {GET_CATEGORY, GET_CATEGORY_SUCCESS} from '../constants';
+
+export function getCategory(params){
+  return {
+    type: GET_CATEGORY,
+    payload: params
+  };
+};
```

Comment:
```suggestion
export function getCategory(payload){
  return {
    type: GET_CATEGORY,
    payload,
  };
};
```
thống nhất tất cả các tham số đầu vào action là payload